.github/pull.yml

```
@@ -0,0 +1,14 @@
+version: "1"
+rules:
+  - base: master
+    upstream: UncleGoogle:master
+    mergeMethod: none  # Optional, one of [none, merge, squash, rebase, hardreset], Default: none.
+    mergeUnstable: false    # Optional, merge pull request even when the mergeable_state is not clean. Default: false
+    # assignees:              # Optional
+    reviewers:              # Optional
+      - urwrstkn8mare
+      - AndrewDWhite
+    conflictReviewers:      # Optional, on merge conflict assign a reviewer
+      - UncleGoogle
+label: ":arrow_heading_down: pull"  # Optional
```

This isn't an important suggestion or maybe even be wrong. Just thought that since we're synchronising with an upstream it and not a normal Pull Request, it should be called sync to differentiate. Although the emoji is already there so feel free to disregard this and resolve it.

```suggestion
label: ":arrow_heading_down: sync"  # Optional
```